weasyprint/css/validation/properties.py

```diff
@@ -89,10 +92,11 @@ def validate_non_shorthand(base_url, name, tokens, required=False):
     if not required and name not in PROPERTIES:
         raise InvalidValues('property not supported yet')
 
-    for token in tokens:
-        var_function = check_var_function(token)
-        if var_function:
-            return ((name, var_function),)
+    if name not in MULTIVAL_PROPERTIES:
```

Comment:
This is necessary because a `content` declaration is a special case since its property handler returns a list, as opposed to every other property. Currently if one of the rules in `content` is a variable function, it would only parse that variable function and return it from the function here, without processing the other rules. This is why I have added a special case condition here.

I cleaned it up by declaring the content property as a multi value property.